geode-core/src/test/java/org/apache/geode/internal/cache/DistributedPutAllOperationTest.java

```diff
@@ -33,4 +47,39 @@ public void shouldBeMockable() throws Exception {
 
     assertThat(mockDistributedPutAllOperation.getBaseEvent()).isSameAs(mockEntryEventImpl);
   }
+
+  @Test
+  public void testDoRemoveDestroyTokensFromCqResultKeys() {
```

Comment:
A small refactoring request...to make it readable, can this be broken down (with new lines) into:
setup
action
validation/verification